plugins/operators/ray_tune/ado_ray_tune/config.py

```diff
@@ -14,6 +14,57 @@
 )
 
 
+def create_optuna_ray_tune_config(
+    metric: str | list, mode: str | list, parameters: dict, tune_options: dict
+) -> ray.tune.TuneConfig:
+    try:
+        from ray.tune.search.optuna import OptunaSearch
+    except ImportError:
+        raise ImportError("Optuna must be installed! Run `pip install optuna`.")
+
+    # Params dict is already preprocessed (sampler class instantiated) by model validator
+    search_alg = OptunaSearch(metric=metric, mode=mode, **parameters)
+    tune_options["search_alg"] = search_alg
+    tune_options["metric"] = metric[0] if isinstance(metric, list) else metric
+    tune_options["mode"] = mode[0] if isinstance(mode, list) else mode
+    return ray.tune.TuneConfig(**tune_options)
+
+
+def create_lhc_ray_tune_config(
```

Comment:
Everywhere else we seem to refer to this as LHU - it could be worth changing the function name to be lhu as well